server/index.js

```diff
@@ -30,6 +36,34 @@ app.get('/description', (req, res) => {
   });
 });
 
+app.post('/post', (req, res) => {
+  const post = req.body.postItem[0];
+  // req.setTimeout(0);
+  db.saveOne(post, (err) => {
+    if (err) { throw err; }
+    res.status(200);
+  });
+});
+
+app.delete('/delete', (req, res) => {
```

Comment:
This is just typical REST API nomenclature, but you'll probably want your delete call to have the ID of the item present in the URL. E.g. `/delete/:id`, where `:id` is the ID of the item in the database.

Check out e.g. the IMGUR API where you delete in this same way: https://api.imgur.com/endpoints/image/#image-delete

Reply:
Yeah, your suggestion fits a more "real-world" scenario. I'll make that change.